diff --git a/build_FFmpeg/build.sh b/build_FFmpeg/build.sh
index ef82b48..0a73a84 100644
--- a/build_FFmpeg/build.sh
+++ b/build_FFmpeg/build.sh
@@ -49,22 +49,15 @@ rm -f "$TESTFILE"
 
 rm -rf ffbuild
 mkdir ffbuild
-
-FFMPEG_REPO="${FFMPEG_REPO:-https://github.com/FFmpeg/FFmpeg.git}"
-FFMPEG_REPO="${FFMPEG_REPO_OVERRIDE:-$FFMPEG_REPO}"
-GIT_BRANCH="${GIT_BRANCH:-master}"
-GIT_BRANCH="${GIT_BRANCH_OVERRIDE:-$GIT_BRANCH}"
+unzip ../third_party/third_party_sources/FFmpeg-release-4.4.zip -d ./ffbuild/
+mv ./ffbuild/FFmpeg-release-4.4 ./ffbuild/ffmpeg
 
 BUILD_SCRIPT="$(mktemp)"
 trap "rm -f -- '$BUILD_SCRIPT'" EXIT
 
 cat <<EOF >"$BUILD_SCRIPT"
     set -xe
-    cd /ffbuild
-    rm -rf ffmpeg prefix
-
-    git clone --filter=blob:none --branch='$GIT_BRANCH' '$FFMPEG_REPO' ffmpeg
-    cd ffmpeg
+    cd /ffbuild/ffmpeg
 
     ./configure --prefix=/ffbuild/prefix --pkg-config-flags="--static" \$FFBUILD_TARGET_FLAGS $FF_CONFIGURE \
         --extra-cflags='$FF_CFLAGS' --extra-cxxflags='$FF_CXXFLAGS' \
